src/03_modules.py

```diff
@@ -5,27 +5,35 @@
 level operating system functionality.
 """
 
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+listOfArguments = sys.argv[0].split('/')
```

Comment:
Your solution works, but you can skip this step.
`for argument in sys.argv:` would work the same.